content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -28,6 +41,18 @@ export class _StartupOverlay extends React.PureComponent {
     }, 10);
   }
 
+  async setFxaMetrics() {
```

Comment:
Again, no need for a different function here, just adds an extra bind in the constructor that you don't need.